neuralmonkey/run.py

```diff
@@ -29,6 +29,7 @@
 CONFIG.ignore_argument('validation_period')
 CONFIG.ignore_argument('logging_period')
 CONFIG.ignore_argument('minimize')
+CONFIG.ignore_argument('random_seed')
```

Comment:
Nevím co je minimum risk training, ale podle názvu je mi podezřelé, že to dělá něco s `run`. V současném `run` se to taky nenastavuje, ale můžu to tam přidat.